site/en/docs/privacy-sandbox/user-agent/index.md

```diff
@@ -50,32 +54,32 @@ JavaScript.
 to the full set of user-agent data, but only when servers actively declare an
 explicit need for specific pieces of data.
 
-By removing passively exposed user-data, we can better measure and limit
-[how much information is intentionally exposed](https://github.com/bslassey/privacy-budget)
-(by request headers, JavaScript APIs, and other mechanisms).
+By removing passively exposed user-data, we can better measure and reduce the
+amount of information that is intentionally exposed by request headers,
+JavaScript APIs, and other mechanisms.
 
 ## Why do we need reduced UA and UA-CH?
 
 Currently, the User-Agent string broadcasts a large string of data about a
 user's browser, operating system, and version every HTTP request. This is
-problematic because:
+problematic for two reasons:
 
-*  the granularity and abundance of detail can lead to user identification;
-*  and the default availability of this information can lead to covert tracking.
+*  The granularity and abundance of detail can lead to user identification.
+*  The default availability of this information can lead to covert tracking.
 
-We improve user privacy by only sharing basic information.
+We improve user privacy by only sharing basic information by default.
 
 The reduced User-Agent includes the browser's brand and a significant version,
-desktop or mobile, and the platform. To access more data, User-Agent Client
-Hints allow you to request specific information about the user's device or
-conditions in a way that can be audited. 
+where the request came from (desktop or mobile), and the platform. To access
+more data, User-Agent Client Hints allow you to request specific information
+about the user's device or conditions. 
```

Comment:
I know 'conditions' is used in [web.dev/user-agent-client-hints](https://web.dev/user-agent-client-hints/), but I wonder if 'device attributes' (or whatever) might be better — assuming that's what this refers to.